LayoutTests/compositing/shared-backing/bug276216.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
I don't like test cases with bug numbers in the name because it's hard to know what they do. Let's call it `update-backing-sharing-on-size-change.html` or something.

Reply:
If this fixes https://bugs.webkit.org/show_bug.cgi?id=275937 please also make a test case from its attachment and include it here.